authentik/providers/proxy/tasks.py

```diff
@@ -23,8 +23,10 @@ def proxy_set_defaults():
 
 
 @CELERY_APP.task()
-def proxy_on_logout(session_id: str):
+def proxy_on_logout(session_id: str | None):
```

Comment:
The reason why this wasn't handled is because it should never happen, I'll need to do some digging to see the reason this happens